include/SSVOpenHexagon/Data/StyleData.hpp

```diff
@@ -7,6 +7,7 @@
 #include "SSVOpenHexagon/Global/Common.hpp"
 #include "SSVOpenHexagon/Data/ColorData.hpp"
 #include "SSVOpenHexagon/Data/CapColor.hpp"
+#include "SSVOpenHexagon/Core/HGStatus.hpp"
```

Comment:
We can avoid a physical dependency on this header file by forward-declaring `HexagonGameStatus`. This usually speeds up compilation.